android/gltfio-android/src/main/java/com/google/android/filament/gltfio/MaterialProvider.java

```diff
@@ -70,6 +70,12 @@ public MaterialKey() {}
             nGlobalInit();
         }
         private static native void nGlobalInit();
+
+        public void constrainMaterial(@NonNull @Size(min = 8) int[] uvmap) {
```

Comment:
Please keep the static native methods last like we do in the other JNI bindings. Also, could you add a documentation block to explain what this API is for?

Reply:
`constrainMaterial` has an explanation in native code, it's just a wrapper. Should I copy it over?

Reply:
That's usually what we do yes. More importantly the doc should mention why a user of gltfio should care or use this API.